csm/common/errors.py

```diff
@@ -119,31 +134,55 @@ class CsmPermissionDenied(CsmError):
     This error is raised for all cases when we don't have permissions
     """
 
-    def __init__(self, desc=None, message_id=None, message_args=None):
+    _desc = "Access to the requested resource is forbidden"
+
+    def __init__(self, desc=None, message_id=const.PERMISSION_DENIED_ERROR, message_args=None):
         super(CsmPermissionDenied, self).__init__(
-            CSM_INTERNAL_ERROR, desc,
+            CSM_OPERATION_NOT_PERMITTED, desc,
             message_id, message_args)
 
 
 class CsmResourceNotAvailable(CsmInternalError):
 
     """Describes issues when requested resource is not available"""
 
+    _desc = "Requested resource is not available"
```

Comment:
Requested resource is not available --> Resource not available

Reply:
For this error, message and message id will be similar 
message = "Resource not available"
message_id = "ResourceNotAvailable"
is it ok?